contracts/protocol/OtcMarketCreateOffer.sol

```diff
@@ -75,6 +74,8 @@ abstract contract OtcMarketCreateOffer is IOtcMarketCreateOffer, OtcMarketCore {
         createOfferReceipt = CreateOfferReceipt(offerId, srcAmountLD);
 
         Transfer.transferFrom(srcTokenAddress, address(escrow), srcAmountLD);
```

Comment:
Reentrancy here.

State changes before transfer:
1. Offer is created and added to the mapping
2. `OfferCreated` message is sent